app/pb_kits/playbook/pb_caption/_caption.html.erb

```diff
@@ -1,5 +1,7 @@
-<%= content_tag(object.tag, object.text,
+<%= content_tag(object.tag,
+    aria: object.aria,
     id: object.id,
     data: object.data,
-    class: object.classname,
-    aria: object.aria) %>
+    class: object.classname) do %>
+    <%= object.children.present? ? capture(&object.children) : object.text %>
```

Comment:
@christinaatai and I discussed this and we thought it was a reasonable idea to create a block version of `Caption`. Hoping for some special attention here since my Ruby knowledge is basic. I based it off of how the [Body Kit](https://github.com/powerhome/playbook/blob/NUX-639/time-kit-resize/app/pb_kits/playbook/pb_body/_body.html.erb) handles it.

Reply:
By making this into a block kit, we can pass icons through it.